package NDnnProto;


message TSpikeNeuronInnerState {
    optional double Membrane = 1;
    optional double SpikeProbability = 2;
    optional double ProbabilityModulation = 3;
    optional bool Fired = 4;
}

